project/Dependencies.scala

```diff
@@ -4,21 +4,21 @@ import sbt.Keys._
 object Dependencies {
 
   // Version numbers
-  val PlayVersion = "2.5.13"
-  val AkkaVersion = "2.4.17"
+  val PlayVersion = "2.6.0-M3"
```

Comment:
2.6.0-R1 is out now. If you upgrade to it you will no longer need Lagom's build-link project and hopefully not its `Reloader` either